src/components/results/shortcircuit/shortcircuit-analysis-all-buses-result.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import {
+    SCAFaultResult,
+    SCAFeederResult,
+    ShortCircuitAnalysisType,
+} from 'components/results/shortcircuit/shortcircuit-analysis-result.type';
+import { ShortCircuitAnalysisResult } from 'components/results/shortcircuit/shortcircuit-analysis-result';
+import { useSelector } from 'react-redux';
+import { ReduxState } from 'redux/reducer.type';
+import { useCallback, useState } from 'react';
+import { ComputingType } from 'components/computing-status/computing-type';
+
+export const ShortCircuitAnalysisAllBusesResult = () => {
+    const allBusesShortCircuitNotif = useSelector(
+        (state: ReduxState) => state.allBusesShortCircuitNotif
+    );
+
+    const allBusesShortCircuitAnalysisStatus = useSelector(
+        (state: ReduxState) =>
+            state.computingStatus[ComputingType.ALL_BUSES_SHORTCIRCUIT_ANALYSIS]
+    );
+
+    const [result, setResult] = useState<SCAFaultResult[]>([]);
+
+    const updateResult = useCallback(
```

Comment:
You don't have a useEffect which fetchShortCircuitAnalysisResult here ? for ALL_BUSES, mode NONE and all you know ?

Reply:
Nope because it was not needed here, as it was done before my PR.
But as discussed maybe the best thing to do is call this endpoint for both one and all buses.